src/expectRevert.js

```diff
@@ -0,0 +1,59 @@
+const { web3 } = require('./setup');
+
+const { expect } = require('chai');
+const colors = require('ansi-colors');
+const semver = require('semver');
+
+async function expectException (promise, expectedErrors) {
+  try {
+    await promise;
+  } catch (error) {
+    for (const expectedError of expectedErrors) {
+      expect(error.message).to.include(expectedError, `Wrong failure type, expected '${expectedError}'`);
+    }
+    return;
+  }
+
+  throw Error('Expected failure not received');
+}
+
+const expectRevert = async function (promise, expectedError) {
+  if (!expectedError) {
+    try { await promise; } catch (error) { }
```

Comment:
We need to await this here, but I'm not sure if there's a more idiomatic way of doing this. @frangio?

Reply:
We don't need to await it, only to make sure that an exception is caught. We can do this as follows:

```suggestion
    try { await promise; } catch (error) { }
    promise.catch(() => { });
```